client/src/plugins/camunda-plugin/deployment-tool/DeploymentTool.js

```diff
@@ -194,45 +210,39 @@ export default class DeploymentTool extends PureComponent {
     return configuration;
   }
 
+  removeCredentials = async () => {
+    const savedConfiguration = await this.getSavedConfiguration(this.state.activeTab);
+    const omited = omit(savedConfiguration.endpoint, ['username', 'password', 'token']);
+    this.saveEndpoint({
+      ...omited,
+      rememberCredentials: false
+    });
+  }
+
+  saveCredential = async (credential) => {
+    const savedConfiguration = await this.getSavedConfiguration(this.state.activeTab);
+    this.saveEndpoint({
+      ...savedConfiguration.endpoint,
+      rememberCredentials: true,
+      ...credential
+    });
+  }
+
   async saveEndpoint(endpoint) {
 
     const {
-      id,
-      url,
-      authType,
-      rememberCredentials,
-      username,
-      password,
-      token
+      rememberCredentials
     } = endpoint;
 
-    const authConfiguration =
-      authType === AuthTypes.none
-        ? {}
-        : authType === AuthTypes.basic
-          ? {
-            username,
-            password: rememberCredentials ? password : ''
-          }
-          : {
-            token: rememberCredentials ? token : ''
-          };
-
-    const endpointConfiguration = {
-      id,
-      url,
-      authType,
-      rememberCredentials,
-      ...authConfiguration
-    };
+    const endpointToSave = rememberCredentials ? endpoint : omit(endpoint, ['username', 'password', 'token']);
```

Comment:
```suggestion
    const endpointToSave = rememberCredentials ? endpoint : omit(endpoint, [ 'password', 'token' ]);
```
(username was to be preserved even without `rememberCredentials=true`)

Reply:
This is not wanted anymore as stated by new designs.